config/manager/manager-service.yaml

```diff
@@ -3,11 +3,15 @@ kind: Service
 metadata:
   name: service
   labels:
-    control-plane: controller-manager
-    app.kubernetes.io/created-by: data-science-pipelines-operator
+    app.kubernetes.io/name: service
```

Comment:
I think this is too granular:
The [recommended labels](https://kubernetes.io/docs/concepts/overview/working-with-objects/common-labels/) in Kubernetes are supposed to be about application, not individual resources (otherwise they'd be redondant).
Here we would rather have either "controller-manager" (if we consider the controller as an application itself, part of DSPO), or just "DSPO" / "data-science-pipelines-operator".

Reply:
Makes sense, would you suggest the name label be just "data-science-pipelines-operator" ?

Reply:
I think so. In that case I we consider the application is "data-science-pipelines-operator" which is itself "part-of" ODH.

The other alternative: the overall product ("part-of") is DSPO, but in that case I'm not sure what should be in "name".
-> which probably hints that the first alternative is better.